xml/yast2_gui.xml

```diff
@@ -0,0 +1,144 @@
+ <primary>&yast;</primary></indexterm>
+ <para>
+  &yast; is the installation and configuration tool for &productname;.
+  It has an easy to use graphical interface and with the capability to
+  customize your system quickly during and after the installation. It can
+  be used to setup hardware, configure the network, system services and
+  tune your security settings.
+ </para>
+ <sect1 xml:id="sec.yast.gui.advanced_hotkeys">
+  <title>Advanced Hotkeys</title>
+  <para>&yast; has a set of advanced hotkeys.</para>
+  <variablelist>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap>Print Screen</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Take and save a screenshot. May not be available when &yast; is running under
+      some desktop environments.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="shift"/><keycap>F4</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Enable / disable the color palette optimized for vision impaired users.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="shift"/><keycap>F7</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Enable / disable logging of debug messages.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="shift"/><keycap>F8</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Open a file dialog to save log files to a non-standard location.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="control"/><keycap function="shift"/><keycap function="alt"/><keycap>D</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Send a DebugEvent. &yast; modules can react on this by executing special debugging actions.
+      The result depends on the specific &yast; module.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="control"/><keycap function="shift"/><keycap function="alt"/><keycap>M</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Start / stop macro recorder.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="control"/><keycap function="shift"/><keycap function="alt"/><keycap>P</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Replay macro.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     <keycombo action="press"><keycap function="control"/><keycap function="shift"/><keycap function="alt"/><keycap>S</keycap></keycombo>
+    </term>
+    <listitem>
+     <para>
+      Show style sheet editor.
```

Comment:
"stylesheet" as one word? not sure whether I am just saying that because I have too much to do with CSS.